src/config_file.cc

```diff
@@ -189,6 +189,35 @@ void CONFIG_TREC::record_tag( string config_tag, string config_value ) {
 	}
   }
   
+  if (config_tag == "bm25plusparam_k1" ) {
+	bm25plusparam_k1 = strtod(config_value.c_str(),NULL);
+	found = 1;
+  }
+
+  if (config_tag == "bm25plusparam_k2" ) {
+	bm25plusparam_k2 = strtod(config_value.c_str(),NULL);
+	found = 1;
+  }
+
+  if (config_tag == "bm25plusparam_k3" ) {
```

Comment:
I don't have bm25plus in my master. Is it merged in master of xapian.

if not can you pass me the patch file which i can apply and compile the evaluation module with BM25Plus ?